src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -229,12 +234,21 @@ def course_enrollment_widget_props(context):
     """
     course_run = context["run"]
 
+    starts_in_message = None
+    if course_run.start > timezone.now():
+        course_start = arrow.get(course_run.start)
+        humanized_course_start = course_start.humanize(
+            arrow.now(), locale=to_locale(get_language())
+        )
+        starts_in_message = f"{_('The course will start')} {humanized_course_start}"
```

Comment:
Having doubts whether we can presume that order will be the same in every language... If not this is safer:
```suggestion
        starts_in_message = _("The course will start {:s}'').format(humanized_course_start)
```